src/op_sync_team.rs

```diff
@@ -202,14 +211,18 @@ pub fn sync_team(
             }
         }
         match action {
-            TeamAction::Add(github_name, handle) => {
+            TeamAction::Add(github_name, github_id, handle) => {
                 let logger = logger.new(o!(
                     "nixpkgs-handle" => format!("{}", handle),
                     "github-name" => format!("{}", github_name),
                 ));
+
                 if pending_invites.contains(&github_name) {
                     noops.inc();
                     debug!(logger, "User already has a pending invitation");
+                } else if invited.contains(&github_id) {
+                    noops.inc();
+                    debug!(logger, "User was already invited previously (since there's no pending invitation we can assume the user rejected the invite)");
```

Comment:
Previously we tracked "noops" as a proxy for how many people were pending invitations. Can you add two metrics for this,

below:

```
    let removals =
        register_int_counter!("rfc39_team_sync_removals", "Total team removals").unwrap();
```

```
    let pending_invitations =
        register_int_counter!("rfc39_team_sync_invite_pending", "Total pending team invitations pending").unwrap();
    let previously_invited =
        register_int_counter!("rfc39_team_sync_previously_invited", "Total users not invited because we know we invited them already").unwrap();
```

and then:

```suggestion
                if pending_invites.contains(&github_name) {
                    noops.inc();
                    pending_invitations.inc();
                    debug!(logger, "User already has a pending invitation");
                } else if invited.contains(&github_id) {
                    noops.inc();
                    previously_invited.inc();
                    debug!(logger, "User was already invited previously (since there's no pending invitation we can assume the user rejected the invite)");
```

and finally extend this bit of code:

```
        let logger = logger.new(o!(
            "dry-run" => dry_run,
            "github-id" => format!("{}", github_id),
            "changed" => additions.get() + removals.get(),
            "additions" => additions.get(),
            "removals" => removals.get(),
            "noops" => noops.get(),
            "errors" => errors.get(),
        ));
```

to include these two metrics?